docs/advanced/subclassed-layers.md

```diff
@@ -59,17 +59,22 @@ export default MultiColorPathLayer extends PathLayer {
 
 You can replace the shaders used in a layer by overriding the `getShaders()`
```

Comment:
Could we add any links to luma.gl shader modules docs (or maybe they are still pretty thin?)